WordPress/src/main/java/org/wordpress/android/ui/reader/adapters/ReaderPostAdapter.java

```diff
@@ -311,7 +311,7 @@ private void renderTagHeader(
                 onFollowButtonClicked,
                 ReaderTagTable.isFollowedTagName(currentTag.getTagSlug()),
                 isFollowButtonEnabled,
-                AppPrefs.isReaderImprovementsPhase2Enabled() || mAccountStore.hasAccessToken()
+                mAccountStore.hasAccessToken()
```

Comment:
Tag header changes were released before RI FF flag was removed [here](https://github.com/wordpress-mobile/WordPress-Android/pull/12689). At that time we did not allow following tags locally for the logged out users. We can remove this access token check now.

Reply:
Ohh nice catch! Thanks 🙇 

Fixed in dfd1c7101cfb52ffe782f0ec1f9aa32fe1f93f19